sound/soc/sof/intel/hda-dai.c

```diff
@@ -190,7 +190,8 @@ static int hda_link_dma_hw_params(struct snd_pcm_substream *substream,
 	return 0;
 }
 
-static int hda_dai_hw_free(struct snd_pcm_substream *substream, struct snd_soc_dai *cpu_dai)
+static int __maybe_unused hda_dai_hw_free(struct snd_pcm_substream *substream,
```

Comment:
type in commit message: 'defined **but** not used'

Reply:
fixed, thanks